package.json

```diff
@@ -18,11 +18,14 @@
     "build": "wireit",
     "build:ts": "wireit",
     "build:lit-starter-ts": "wireit",
-    "format": "npm run format:eslint && npm run format:prettier",
-    "format:eslint": "eslint --ignore-pattern=examples/ --fix \"**/*.{js,ts}\"",
-    "format:prettier": "prettier \"**/*.{cjs,html,js,json,md,ts}\" --write",
+    "fix": "npm run lint:fix && npm run format:fix",
+    "format": "npm run format:fix",
```

Comment:
Nice. I wonder if a good next step would be to run these with wireit so we take advantage of caching. Would that even be worth it considering how long these take? @aomarks ?

Reply:
I think I generally have these working in wireit, but one thing I'm a bit confused about is the `output` files list. My initial guess was that the `:fix` versions of these scripts should use the same glob for `files` and `output` since those scripts both read and potentially write to all of them, but this causes all of the matching files to be deleted. Is `output` only for files that are specifically created by the script, not just any that it might modify?

Reply:
I think it would be overkill to do wireit for these. Wireit probably isn't great for scripts like these where only some files are touched during run but we can't know which ones until running. So instead `files` and `output` here are potentially the entirety of the monorepo. It'll probably end up taking more space in the github cache unnecessarily.

Reply:
Yeah, those globs match almost everything. I'll roll the wireit part back for now and we can add it later again if we find it's useful.